algorithms/FindersKeepers.js

```diff
@@ -0,0 +1,6 @@
+// Create a function that looks through an array (first argument) and returns the first element in the array that passes a truth test (second argument).
+
+function findElement(arr, func) {
+  filtered = arr.filter(func);
```

Comment:
This function goes through the whole array even if the first element passes the truth test. This is going to be inefficient if the array is a large array. Can you change this such that it doesnt go through the whole array?